OpenSource : 5.26 Explain everything related to Linkerd #660

Khushi-Agrawal-1 opened the pull request:
<hr>

## Description 📜

I have added documentation on the Explanation of Linkerd in Section 5 in an open-source folder of the Opensource Branch.
Fixes #660

<hr>

## Type of change 📝

<!----Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this----->

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

<!----Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this----->

- [x] Open Source

<hr>

## Checklist ✅

<!----Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this----->

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<!----Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this----->

## Screenshots / Gif (Optional) 📸
![image](https://user-images.githubusercontent.com/64862672/134037888-56819605-719c-4efd-b651-00247936adc3.png)

<hr>

Aryamanz29 commented on C_CPP/readme.md:
```diff
@@ -0,0 +1 @@
+
```
Delete this line from `C_CPP/readme.md`

Aryamanz29 reviewed (changes requested):
Delete this line from `C_CPP/readme.md`

Khushi-Agrawal-1 commented:
![image](https://user-images.githubusercontent.com/64862672/134039348-f874c4be-d436-40dc-bdd3-8d1234e21bfe.png)

For me its showing that there's nothing written in that file. What should I do?

Aryamanz29 commented:
> ![image](https://user-images.githubusercontent.com/64862672/134039348-f874c4be-d436-40dc-bdd3-8d1234e21bfe.png)
> 
> For me its showing that there's nothing written in that file. What should I do?

Remove that blank line

Khushi-Agrawal-1 commented:
How do I delete the first line? The only option I can see is to delete the file.

Aryamanz29 commented:
> How do I delete the first line? The only option I can see is to delete the file.

Do this through your local environment, then` git add . then git commit, push`

Khushi-Agrawal-1 commented:
![image](https://user-images.githubusercontent.com/64862672/134042140-93192c68-3b9c-4830-ade5-c34dd6989fcc.png)

I deleted the line like how you told. So now its showing that I deleted something when I'm creating a PR. So is this alright?

Khushi-Agrawal-1 commented:
And its only showing this when I'm choosing the base as "main" and not "Opensource"

Khushi-Agrawal-1 commented:
![image](https://user-images.githubusercontent.com/64862672/134045254-9c714a67-1e85-40bb-a592-22a6925f8ee0.png)

Its showing 1 file changed when I choose "main" as base but its showing 2 files changed when I choose "Opensource". What to do?

Khushi-Agrawal-1 commented:
Can I use "main" as the base ?

Aryamanz29 commented:
> Can I use "main" as the base ?

No, Please create PR from `your_fork : Opensource Branch to girlscript : Opensource Branch`
